README.md

```diff
@@ -85,6 +84,32 @@ Then build the target release with the following command:
 cargo build --release
 ```
 
+### Test
+
+Use the following command to run tests:
+
+```sh
+cargo test
+```
+
+To run tests with benchmarks
+```sh
+cargo test --features runtime-benchmarks
+```
+
+### Benchmark
+
+Use the following command to run the benchmark, use the desired pallet name as required:
+
+```sh
+cargo run --release --features=runtime-benchmarks -- benchmark --chain dev  --execution=wasm --wasm-execution compiled --extrinsic="*" --pallet=<pallet_name> --steps=20 --repeat=1 --heap-pages=4096 --output .
```

Comment:
I know our `Makefile` might need some love and updating to be more useful, currently it's a bit complex, but for this is going to be quite useful. Instead of a long command like this one let's just copy it in the Makefile and have a `make bench` or similar that would run this in a more convenient way.